tools/build_go_exe.sh

```diff
@@ -30,19 +30,13 @@ fi
 GO_VERSION=$(go version | awk '{print $3}')
 UPX_BINARY="" # only enabled for go1.7+
```

Comment:
Now that the go-version-specific UPX checks are gone, the `UPX_BINARY` setting at L34 can be pulled up here. It'd probably make sense to leave a comment along the lines of `UPX only works on binaries produced by go1.7+, but we require go1.8+`.

Previously: (Is UPX installed PATH) AND (Is go 1.7+)
Now: (Is UPX installed in PATH)